src/ambiguity_test.c

```diff
@@ -585,8 +585,8 @@ s8 make_dd_measurements_and_sdiffs(u8 ref_prn, u8 *non_ref_prns, u8 num_dds,
  * to date, this will make a set of sdiffs and DD measurements that correspond
  * to the resolved sats.
  *
- * Returns 0 if the input sdiff sats are a superset of the resolved IAR sats.
- * Returns -1 otherwise.
```

Comment:
I would recommend explicitly stating the return codes of the function rather than referencing the documentation of a different function.